examples/frontend/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "frontend"
```

Comment:
Maybe we should rename it to `demo-webapp`?

Reply:
Not a bad idea